content/en/graphing/metrics/explorer.md

```diff
@@ -11,21 +11,56 @@ further_reading:
     text: "Metrics Distributions"
 ---
 
-Use the [Metrics Explorer page][1] to begin examining the [metrics][2] collected from your application by Datadog.
+## Overview
 
-Clicking into the "Graph" text box brings up a list of all the metrics that Datadog gathers from your application. You can also search the list by typing into the box. Selecting metrics populates real-time graph visualizations on the right hand side. By default, up to 20 graphs are shown. You can change this number by clicking on the "Options" gear.
+The [Metrics Explorer][1] is a basic interface for examining your metrics in Datadog. For more advanced options, create a [notebook][2], [screenboard][3], or [timeboard][4].
```

Comment:
The [Metrics Explorer][1] is a basic interface for examining your metrics in Datadog. For more advanced options, create a [notebook][2] or dashboards ([screenboard][3], or [timeboard][4]).